### PR TITLE
Create zh_CN.lang(s)

### PR DESCRIPTION
**Pull request by SihenZhang**
Before merging, I want to let my friend @TartaricAcid and @3TUSK to have a check.

So sorry because I'm not good at ExtraTrees and Botany, I can't translate them correctly.

### CODE REVIEW

File: core/src/main/resources/assets/binniecore/lang/zh_CN.lang [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+binniecore.machine.tooltips.processing=正在处理
+
+binniecore.errors.task.cancelled.desc=任务取消
+binniecore.errors.task.cancelled.info=由 Buildcraft 的门取消
+binniecore.errors.task.paused.desc=暂停
+binniecore.errors.task.paused.info=由 Buildcraft 的门暂停
+
+binniecore.errors.unknown.name=未知
+binniecore.errors.unknown.desc=这个错误已损坏，请将此反馈给mod作者
```
Comment (3TUSK) [inline core/src/main/resources/assets/binniecore/lang/zh_CN.lang]: `这个错误已损坏`？

File: core/src/main/resources/assets/binniecore/lang/zh_CN.lang [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+binniecore.gui.fieldkit.title=实地考察工具包
+binniecore.gui.fieldkit.info=用于在野外分析蜜蜂、树、花和蝴蝶的工具包。记得带上纸来记笔记。
+binniecore.gui.fieldkit.paper=纸
+binniecore.gui.fieldkit.paper.no=没有纸！
+binniecore.gui.fieldkit.individual=个体
+
+< Descriptions used for effects in the registry >
+binniecore.allele.extrabees.effect.acid.desc=溶解石头和泥土
+binniecore.allele.forestry.effectAggressive.desc=进攻玩家
```
Comment (3TUSK) [inline core/src/main/resources/assets/binniecore/lang/zh_CN.lang]: `攻击玩家`？

File: core/src/main/resources/assets/binniecore/lang/zh_CN.lang [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+binniecore.allele.forestry.effectBeatific.desc=缓慢回复玩家生命
+binniecore.allele.forestry.effectCreeper.desc=在附近引发爆炸
+binniecore.allele.extrabees.effect.spawn_creeper.desc=随机生成爬行者
+binniecore.allele.extrabees.effect.blindness.desc=令玩家失明
+binniecore.allele.forestry.effectDrunkard.desc=灌醉玩家
+binniecore.allele.extrabees.effect.ectoplasm.desc=产生外质蛛网状结构
```
Comment (3TUSK) [inline core/src/main/resources/assets/binniecore/lang/zh_CN.lang]: `extrabees.block.ectoplasm.name=胶质网`

File: genetics/src/main/resources/assets/genetics/lang/zh_CN.lang [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+
+genetics.item.sequence.0=极弱
+genetics.item.sequence.1=弱
+genetics.item.sequence.2=略弱
+genetics.item.sequence.3=中等
+genetics.item.sequence.4=强
+genetics.item.sequence.5=极强
+genetics.item.sequence.unknown=未知
+genetics.item.sequence.unsequenced=未测序
+genetics.item.sequence.partially=部分测序 (%s%)
```
Comment (3TUSK) [inline genetics/src/main/resources/assets/genetics/lang/zh_CN.lang]: `%s%%` or game crashes with unknown formatter

File: genetics/src/main/resources/assets/genetics/lang/zh_CN.lang [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+genetics.item.sequence.sequenced=已测序
+
+bee.modifier.format.max=%.2fx (最大值： %.2fx)
```
Comment (3TUSK) [inline genetics/src/main/resources/assets/genetics/lang/zh_CN.lang]: Unnecessary extra space

**Review by 3TUSK — CHANGES_REQUESTED** [review]

File: core/src/main/resources/assets/binniecore/lang/zh_CN.lang (hunk 1 above)
Comment (SihenZhang) [reply core/src/main/resources/assets/binniecore/lang/zh_CN.lang]: This error is broken.

机器会提示因为什么原因而不能运行，很明显这条就是用于一些非正常情况的

File: core/src/main/resources/assets/binniecore/lang/zh_CN.lang (hunk 2 above)
Comment (SihenZhang) [reply core/src/main/resources/assets/binniecore/lang/zh_CN.lang]: 这是按照旧版翻译直接过来的，到时候改一下吧

File: genetics/src/main/resources/assets/genetics/lang/zh_CN.lang (hunk 4 above)
Comment (SihenZhang) [reply genetics/src/main/resources/assets/genetics/lang/zh_CN.lang]: 然而en_US.lang就是这样的

Comment (3TUSK) [reply genetics/src/main/resources/assets/genetics/lang/zh_CN.lang]: @mezz I believe that `%s)` can [crash the game](https://docs.oracle.com/javase/8/docs/api/java/util/UnknownFormatConversionException.html)... will that even work?

en_US.lang:
https://github.com/ForestryMC/Binnie/blob/bbb91274348db32491a6c7a2b645dd5321903f99/genetics/src/main/resources/assets/genetics/lang/en_US.lang#L223

**Review by TartaricAcid — APPROVED** [review]

File: core/src/main/resources/assets/binniecore/lang/zh_CN.lang (hunk 1 above)
Comment (3TUSK) [reply core/src/main/resources/assets/binniecore/lang/zh_CN.lang]: 未知错误，请反馈给 mod 作者？

**Review by 3TUSK — APPROVED** [review]
This can be merged now.

**Comment by mezz** [comment]
Thank you!

**Pull request merged**